scripts/build.mjs

```diff
@@ -111,10 +111,17 @@ const targets = {
     await cp("./static/icons", "./dist/safari");
 
     rem(
-      "Copying static consent pages to Firefox dist directory"
+      "Copying static pages to Firefox dist directory"
     );
     await cp("./static/_locales", "./dist/firefox/_locales");
     await cp("./static/views", "./dist/firefox/views");
+    await cp("./node_modules/webextension-polyfill/dist", "./dist/firefox/views");
+
+    rem(
+      "Copying static pages to Chrome dist directory"
+    );
+    await cp("./static/views", "./dist/chrome/views");
```

Comment:
Do the Firefox consent pages end up in Chrome as well with this?

Reply:
Yes, installed.html+js ends up in the chrome build, but aren't used. This should be fixed.